packages/office-ui-fabric-react/src/components/DetailsList/DetailsList.tsx

```diff
@@ -1,968 +1,15 @@
-import * as React from 'react';
-import * as stylesImport from './DetailsList.scss';
+import { styled } from '../../Utilities';
```

Comment:
If you did a rename of DetalisList.tsx -> DetailsList.base.tsx in a commit,, then create a brand new DetailsList.ts (not tsx) with the styled export, the line history gets preserved and you don't show up on gitblame for the entire file :p